main.py

```diff
@@ -39,26 +39,40 @@ def hello():
 
 
 @app.route('/circles/api/v1.0/users/<int:person_id>/', defaults={'resource': None})
-@app.route('/circles/api/v1.0/users/<int:person_id>/<resource>', methods=['GET'])
+@app.route('/circles/api/v1.0/users/<int:person_id>/<resource>', methods=['GET', 'POST'])
 def get_person(person_id, resource):
-    # Fetch the person
-    person = Person.match(graph, person_id).first()
-    if not person:
-        abort(404, description='Resource not found')
-    if not resource:
-        # Request specific user.
-        return jsonify(person.json_repr())
-
-    # Request specific resource associated with the person
-    if resource not in [CIRCLES, EVENTS, KNOWS]:
-        abort(404, description='Invalid resource specified')
-
-    elif resource == CIRCLES:
-        return jsonify([c.json_repr() for c in person.IsMember])
-    elif resource == EVENTS:
-        return jsonify([e.json_repr() for e in person.InvitedTo])
-    elif resource == KNOWS:
-        return jsonify([k.json_repr() for k in person.Knows])
+    if request.method == 'PUT':
+        content = request.get_json()
```

Comment:
We probably want to do some content validation. e.g. we can only create a person if we're given display name and email.